src/language/providers/utils.ts

```diff
@@ -223,15 +230,16 @@ export function checkSimpleTypeOpen(
   document: vscode.TextDocument,
   position: vscode.Position
 ) {
+  const nsPrefix = getNsPrefix(document, position)
   var lineNum = position.line
   while (lineNum !== 0) {
     const wholeLine = document
       .lineAt(lineNum)
       .text.substr(0, document.lineAt(lineNum).range.end.character)
 
     if (
-      wholeLine.includes('<xs:simpleType') &&
-      !wholeLine.includes('</xs:simpleType') &&
+      wholeLine.includes('<' + nsPrefix + 'simpleType') &&
```

Comment:
Since this is everywhere, why not define an object that can be called like
`wholeLineElt('simpleType')`
which does this 
`wholeLineIncludes('<' + nsPrefix + arg)`
So that the namespace prefix stuff is suppressed.

Reply:
Might be worth doing that as part of the fix for #165. I suspect that's going to require a decent refactor anyways.